### ISSUE TITLE
Doesn't work with Titanium 7.0.0 GA (Android 2.0.0)

### ISSUE THREAD

**Issue by yahya-uddin** [hansemannn/titanium-firebase-core#9]
I have a basic hello world app, running on Titanium SDK: 7.0.0.GA
It's only other dependency for modules is `ti.cloud`.

The only version of Firebase Core that seems to work on this version of the SDK seems to be: https://github.com/hansemannn/titanium-firebase-core/releases/tag/android-2.0.0

(I understand this is a pre-release).

However, this simple application gave the following error on start-up:

```
[ERROR] TiApplication: (main) [12,503] Sending event: exception on thread: main msg:java.lang.UnsatisfiedLinkError: Couldn't load firebase.core from loader dalvik.system.PathClassLoader[De
xPathList[[zip file "/data/app/com.yahyauddin.example-2.apk"],nativeLibraryDirectories=[/data/app-lib/com.yahyauddin.example-2, /vendor/lib, /system/lib]]]: findLibrary returned null; Tita
nium 7.0.0,2017/12/05 13:30,undefined                                                                                                                                                       
[ERROR] TiApplication: java.lang.UnsatisfiedLinkError: Couldn't load firebase.core from loader dalvik.system.PathClassLoader[DexPathList[[zip file "/data/app/com.yahyauddin.example-2.apk"]
,nativeLibraryDirectories=[/data/app-lib/com.yahyauddin.example-2, /vendor/lib, /system/lib]]]: findLibrary returned null                                                                   
[ERROR] TiApplication:  at java.lang.Runtime.loadLibrary(Runtime.java:358)                                                                                                                  
[ERROR] TiApplication:  at java.lang.System.loadLibrary(System.java:526)                                                                                                                    
[ERROR] TiApplication:  at org.appcelerator.kroll.runtime.v8.V8Runtime.loadExternalModules(V8Runtime.java:151)                                                                              
[ERROR] TiApplication:  at org.appcelerator.kroll.runtime.v8.V8Runtime.initRuntime(V8Runtime.java:118)                                                                                      
[ERROR] TiApplication:  at org.appcelerator.kroll.KrollRuntime.doInit(KrollRuntime.java:205)                                                                                                
[ERROR] TiApplication:  at org.appcelerator.kroll.KrollRuntime$KrollRuntimeThread.run(KrollRuntime.java:114)                                                                                
[ERROR] TiApplication:  at org.appcelerator.kroll.KrollRuntime.init(KrollRuntime.java:136)                                                                                                  
[ERROR] TiApplication:  at com.yahyauddin.example.ExampleApplication.onCreate(ExampleApplication.java:47)                                                                                   
[ERROR] TiApplication:  at android.app.Instrumentation.callApplicationOnCreate(Instrumentation.java:1007)                                                                                   
[ERROR] TiApplication:  at android.app.ActivityThread.handleBindApplication(ActivityThread.java:4344)                                                                                       
[ERROR] TiApplication:  at android.app.ActivityThread.access$1500(ActivityThread.java:135)                                                                                                  
[ERROR] TiApplication:  at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1256)                                                                                             
[ERROR] TiApplication:  at android.os.Handler.dispatchMessage(Handler.java:102)                                                                                                             
[ERROR] TiApplication:  at android.os.Looper.loop(Looper.java:136)                                                                                                                          
[ERROR] TiApplication:  at android.app.ActivityThread.main(ActivityThread.java:5017)                                                                                                        
[ERROR] TiApplication:  at java.lang.reflect.Method.invokeNative(Native Method)                                                                                                             
[ERROR] TiApplication:  at java.lang.reflect.Method.invoke(Method.java:515)                                                                                                                 
[ERROR] TiApplication:  at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:779)                                                                                  
[ERROR] TiApplication:  at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:595)                                                                                                     
[ERROR] TiApplication:  at dalvik.system.NativeStart.main(Native Method)
```

How do I fix this issue?

**Comment by err0x1a** [comment]
Same problem here, but with slightly different errors:

```
[ERROR] TiApplication: (KrollRuntimeThread) [36,1393] Sending event: exception on thread: KrollRuntimeThread msg:java.lang.UnsatisfiedLinkError: dalvik.system.PathClassLoader[DexPathList[[zip file "/data/app/com.domain-1/base.apk"],nativeLibraryDirectories=[/data/app/com.domain-1/lib/x86, /data/app/com.domain-1/base.apk!/lib/x86, /vendor/lib, /system/lib]]] couldn't find "libfirebase.core.so"; Titanium 7.0.0,2017/12/05 13:30,undefined
[ERROR] TiApplication: java.lang.UnsatisfiedLinkError: dalvik.system.PathClassLoader[DexPathList[[zip file "/data/app/com.domain-1/base.apk"],nativeLibraryDirectories=[/data/app/com.domain-1/lib/x86, /data/app/com.domain-1/base.apk!/lib/x86, /vendor/lib, /system/lib]]] couldn't find "libfirebase.core.so"
[ERROR] TiApplication: 	at java.lang.Runtime.loadLibrary(Runtime.java:367)
[ERROR] TiApplication: 	at java.lang.System.loadLibrary(System.java:1076)
[ERROR] TiApplication: 	at org.appcelerator.kroll.runtime.v8.V8Runtime.loadExternalModules(V8Runtime.java:151)
[ERROR] TiApplication: 	at org.appcelerator.kroll.runtime.v8.V8Runtime.initRuntime(V8Runtime.java:118)
[ERROR] TiApplication: 	at org.appcelerator.kroll.KrollRuntime.doInit(KrollRuntime.java:205)
[ERROR] TiApplication: 	at org.appcelerator.kroll.KrollRuntime$KrollRuntimeThread.run(KrollRuntime.java:114)
```

**Comment by yahya-uddin** [comment]
Any update on this?

**Comment by hansemannn** [comment]
No worries, it was just a beta-version as documented. I just released the final 2.0.0 version that supports 7.0.0.GA+ and Android 64-Bit.
